scripts/testing/setup-mock-test-env.py

```diff
@@ -370,7 +370,7 @@ def install_pip_packages_to_mock(mock_command, packages):
     cmd = _prepare_command(mock_command)
 
     cmd = _run_cmd_in_chroot(cmd)
-    cmd.append('pip3 install {}'.format(packages))
+    cmd.append('pip3 install --install-option="--install-scripts=/usr/bin" {}'.format(packages))
```

Comment:
Or we could add `/usr/local/bin` to `PATH` of the mock environment, but I couldn't figure out, how to do that.

Reply:
This solution seems to me better because we then don't depend on a mock more than required. It will be little bit easier to switch to the podman if we decide that in the future.